northstar_tests/tests/integration_tests.rs

```diff
@@ -90,16 +90,10 @@ test!(missing_resource_container, {
     let mut runtime = Runtime::launch().await?;
 
     // install test container without resource
-    runtime.install(get_test_container_npk()).could_fail();
-
     // Expect MissingResource Error
-    runtime
-        .start("test_container")
-        .expect_err(api::model::Error::MissingResource(
-            "test_resource".to_owned(),
-        ))?;
-
-    runtime.uninstall("test_container", "0.0.1").expect_ok()?;
+    runtime.install(get_test_container_npk()).expect_err_kind(
+        api::model::Error::MissingResource("test_resource".to_owned()),
+    )?;
```

Comment:
The only thing that is not that nice with `expect_err_kind` is that the enum's value is ignored. Perhaps we can use something more like:

```rust
    let response = runtime.install(get_test_container_npk());
    assert!(matches!(
        response.0,
        Ok(api::model::Response::Err(api::model::Error::MissingResource(_)))
    ));
```
Maybe we leave that for the next iteration of the integration tests.

Reply:
I don't think you can compile that...you need an expression here, not a wildcard identifier

Reply:
That does indeed compile, but the inner field of `ApiResponse` has to be turn public. But we can better leave this for a general refactoring of the tests.

Reply:
nice...the `match!` macro makes this already possible! even better!